functions/v2/package.json

```diff
@@ -12,19 +12,18 @@
     "node": ">=12.0.0"
   },
   "scripts": {
-    "test": "mocha --timeout 30s test/index.test.js",
+    "test": "mocha test/index.test.js",
```

Comment:
Is there a reason this was deleted?

(Did everything pass consistently without it?)

Reply:
This is no longer necessary. The tests will be much faster, and less flaky because they are no longer forking child processes and waiting for ports.